en/docs/install-and-setup/setup/distributed-deployment/product-profiles.md

```diff
@@ -137,6 +137,12 @@ Create an optimized distribution for a particular API-M profile.
      ```
 
 5.  Start the server with the specified profile. 
+    Configuration optimization is one step occurred in profile optimization process. This is occurred by replacing the 
+    deployment.toml file with a pre-configured profile-specific toml file existing in the pack. If 
+    required, we can skip this step from the profile optimization process, via passing additional option 
+    `--skipConfigOptimization`, so that the existing deployment.toml file in the pack will not be overridden. 
+
+    --optimize option is used so that if the pack is in-place updated even after initial optimization,  the product pack would have fetched irrelevant files for this profile. So by this option, the pack will be profile optimized again.
     
```

Comment:
Change the order of these 2 descriptions, `--skipConfigOptimization` and --optimize.  Use should be implied that optimization is going to occur again initially.

Reply:
Updated. Please check.

Thanks.

Reply:
The pack in place is updated after the initial optimization, and the product pack would have fetched irrelevant files for this profile. The `--optimize` option is used to optimize the pack again.

**Please change the above sentence to below. The required meaning is not derived after the change. Sorry just noticed this late**

If the product pack is "in-place updated" using "WSO2 in-place updates tool" (even after initial profile optimization), it would have fetched irrelevant files for this profile. So by this `--optimize` option, the pack will be profile-optimized again and it will make sure that the pack will be in a correctly optimized state always.